conda_smithy/configure_feedstock.py

```diff
@@ -1741,9 +1753,7 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         "build_with_mambabuild": True,
         # feedstock checkout git clone depth, None means keep default, 0 means no limit
         "clone_depth": None,
-        # Specific channel for package can be given with
-        # `${url or channel_alias}::package_name`, defaults to conda-forge channel_alias
```

Comment:
Let's keep this comment

Reply:
Thank you. I added a slightly reformated comment